tests/miniconda_install_test.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+
+DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../scripts/newinstall.sh
+. "${DIR}/../scripts/newinstall.sh"
+
+# "stub" out cmd to prevent curl from being run
+# shellcheck disable=SC2034
+cmd="echo"
+
+# test expected usage
+miniconda::install 3 4.2.12 /dne
```

Comment:
It is a required argument to the {{miniconda::install}} function which is passed on to the miniconda installer.  The value doesn't matter at this point since {{cmd}} has been stubbed out.